src/SprykerEco/Zed/Payone/PayoneConfig.php

```diff
@@ -110,9 +110,6 @@ public function getRequestStandardParameter()
         $standardParameter->setKey($settings[PayoneConstants::PAYONE_CREDENTIALS_KEY]);
         $standardParameter->setPaymentGatewayUrl($settings[PayoneConstants::PAYONE_PAYMENT_GATEWAY_URL]);
 
-        $standardParameter->setCurrency(Store::getInstance()->getCurrencyIsoCode());
-        $standardParameter->setLanguage(Store::getInstance()->getCurrentLanguage());
-
```

Comment:
It is major change
> https://spryker.atlassian.net/wiki/spaces/CORE/pages/3048767586/Is+my+module+change+major

> Change the returned value (default) of module config method (even adding or removing a key from an array)


Please fix in the following way:
Pass store as optional parameter, if it is present - set data from the Store transfer.
Move the currently removed code to deprecated protected method to have a fallback, and add a message that it will be removed when DS is always enabled.